spec/install/test.js

```diff
@@ -58,7 +50,6 @@ function test(file) {
   }
 }
 
-install();
 tsc();
-test('tmp/conf.js');
-test('tmp/typescript_conf.js');
+// test('tmp/conf.js');
```

Comment:
Do these tests not work anymore?

Reply:
The entire install script does not work. We need to rethink this whole thing.